source/screens/caseScreens/CaseSummary.js

```diff
@@ -482,106 +595,84 @@ const CaseSummary = (props) => {
+                                <Card.CalculationRowCell>
+                                  <Text>{formatAmount(reduction.amount)}</Text>
                                 </Card.CalculationRowCell>
                               </Card.CalculationRow>
                             ))}
                           </>
                         ) : (
-                          <Card.Text italic>Det finns inga registrerade normer.</Card.Text>
-                        )}
-
-                        <Card.DetailsTitle type="h5">Reducering</Card.DetailsTitle>
-                        {calculation?.reductions?.reduction ? (
-                          <>
-                            {convertDataToArray(calculation?.reductions?.reduction).map(
-                              (reduction, index) => (
-                                <Card.CalculationRow key={`${index}-${reduction.type}`}>
-                                  <Card.CalculationRowCell>
-                                    <Text>{reduction?.type}</Text>
-                                  </Card.CalculationRowCell>
-                                  <Card.CalculationRowCell>
-                                    <Text>
-                                      {reduction?.days} {reduction?.days > 1 ? 'dagar' : 'dag'}
-                                    </Text>
-                                  </Card.CalculationRowCell>
-                                  <Card.CalculationRowCell>
-                                    <Text>{reduction?.note}</Text>
-                                  </Card.CalculationRowCell>
-                                  <Card.CalculationRowCell>
-                                    <Text>{formatAmount(reduction.amount)}</Text>
-                                  </Card.CalculationRowCell>
-                                </Card.CalculationRow>
-                              )
-                            )}
-                          </>
-                        ) : (
-                          <Card.Text italic>Det finns inga registrerade reduceringar.</Card.Text>
+                          <Card.Text italic>
+                            Det finns inga registrerade reduceringar.
+                          </Card.Text>
                         )}
 
                         <Card.CalculationRow>
                           <Card.Text strong>Summa</Card.Text>
-                          <Card.Text strong>{formatAmount(calculation.calculationsum)}</Card.Text>
+                          <Card.Text strong>
+                            {formatAmount(calculation.calculationsum)}
+                          </Card.Text>
                         </Card.CalculationRow>
 
-                        <Card.DetailsTitle type="h5">Dokumentation</Card.DetailsTitle>
+                        <Card.DetailsTitle type="h5">
+                          Dokumentation
+                        </Card.DetailsTitle>
 
                         {journals?.journal?.notes?.note?.length > 0 ? (
```

Comment:
This journal display code (inside the calculations detail) should be removed as according to the design, as otherwise journals will be shown twice when details are expanded.